atc/utils/stringencrypt.go

```diff
@@ -0,0 +1,95 @@
+package utils
```

Comment:
Guessing this whole thing can be removed once everything's switched over to the preexisting encryption stack. :)